extensions/blocks/dialogue/components/speakers-control.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { map } from 'lodash';
```

Comment:
Do we need to use the lodash implementation? What advantage does it give us over Array.map?

Reply:
🤷 some folks use lodash, some don't. I can replace it with the native method. I think it was more relevant a long time ago, where lodash acted as a compatibility and support layer.

Reply:
we are using lodash in multiple files, so I think we can keep using it. If we agree to remove it, I'll glad to do it in a follow-up PR? Marking it as resolved.